packages/react/src/ErrorBoundary.en.mdx

```diff
@@ -48,6 +48,38 @@ const ErrorAfter4s = () => {
 
 ![Example banner](/gif/errorboundary-example.gif)
 
+:::tip
+
+## Use `<ErrorBoundary/>` fallback props without prop drilling
+
+### useErrorBoundaryFallbackProps
+
+In fallback of `<ErrorBoundary/>`, by calling useErrorBoundaryFallbackProps().reset/error, we can use error of `<ErrorBoundary/>` and reset it without using prop directly
+
+```tsx
+import { ErrorBoundary, useErrorBoundaryFallbackProps } from '@suspensive/react'
+
+const Nested = () => {
+  const props = useErrorBoundaryFallbackProps()
+
+  return (
+    <>
+      <button onClick={props.reset} />
```

Comment:
```suggestion
      <button onClick={props.reset}>click to reset error!</button>
```

how about like this?

Reply:
I updated it in [1e748f8](https://github.com/suspensive/react/pull/216/commits/1e748f837c19d8e3cd4214e363fece8e62ddfc6e)

Almost cases we use text as `Try again` to reset ErrorBoundary